elixir/apps/web/lib/web/live/policies/edit.ex

```diff
@@ -48,9 +48,11 @@ defmodule Web.Policies.Edit do
               phx-debounce="300"
             />
             <:actions>
-              <.button phx-disable-with="Updating Policy..." class="w-full">
-                Update Policy
-              </.button>
+              <div class="w-full flex justify-end">
```

Comment:
probably not needed as default `display: block` of `div` already takes full width:
```suggestion
              <div class="flex justify-end">
```

Reply:
<img width="743" alt="Screenshot 2023-12-11 at 11 19 32 AM" src="https://github.com/firezone/firezone/assets/167144/8efd11ad-156d-4a68-9198-d18dffcc781c">

Yeah but simple_form wraps the `:action` in a "flex justify-between" to distribute the spacing, so I wrapped it with another div to fill the full space and reset to `justify-end`. I considered modifying `simple_form` component but thought this worked for now

Reply:
I'll see if I can DRY it up

Reply:
Fixed with a simple condition to use `justify-between` for multiple actions and `justify-end` for single actions